modules/services/mopidy.nix

```diff
@@ -0,0 +1,67 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.mopidy;
+
+in
+
+{
+  options = {
+    services.mopidy = {
+
+      enable = mkOption {
+        type = types.bool;
+        default = false;
+        description = "Whether to enable the Mopidy Daemon.";
+      };
+
+      package = mkOption {
+        type = types.path;
+        default = "/usr/local";
+        description = "This option specifies the mopidy package to use.";
+      };
+
+      mediakeys = {
+        enable = mkOption {
+          type = types.bool;
+          default = false;
+          description = "Whether to enable the Mopidy OSX Media Keys support daemon.";
+        };
+
+        package = mkOption {
+          type = types.path;
+          default = "/usr/local";
+          description = "This option specifies the osxmpdkeys package to use.";
+        };
+
+      };
+
+    };
+  };
+
+  config = mkMerge [
+    (mkIf cfg.enable {
+      launchd.user.agents.mopidy = {
+        serviceConfig.ProgramArguments = [
```

Comment:
You can use [`serviceConfig.Program`](https://github.com/LnL7/nix-darwin/blob/master/modules/launchd/launchd.nix#L103-L111) if you don't need arguments.